locales/en/email-strings.ftl

```diff
@@ -129,3 +129,26 @@ more-products-vpn = Protection for your whole device, on every device.
 more-products-cta-vpn = Get {-product-name-vpn}
 more-products-relay = Hide your real email address to help protect your identity.
 more-products-cta-relay = Get {-product-name-relay}
+
+## 2022 email template. HTML tags should not be translated, e.g. `<a>`
+
+email-2022-unsubscribe = You’re receiving this automated email as a subscriber of Firefox Monitor. <br>Feel free to change your email preferences at any time <a href="{ $unsubscribe-url }">here</a>.
```

Comment:
```suggestion
email-2022-unsubscribe = You’re receiving this automated email as a subscriber of { -product-name }. <br>Feel free to change your email preferences at any time <a href="{ $unsubscribe-url }">here</a>.
```

Reply:
Do we really need to expose attributes (`href`, `rel`)? Previous strings didn't ([example](https://github.com/mozilla/blurts-server/blob/f8ceec0e35aa999ee81a4ac255035f9ca09e928c/locales/en/email-strings.ftl#L98))

Reply:
> Do we really need to expose attributes (`href`, `rel`)? Previous strings didn't ([example](https://github.com/mozilla/blurts-server/blob/f8ceec0e35aa999ee81a4ac255035f9ca09e928c/locales/en/email-strings.ftl#L98))

@flodolo true, but the resulting HTML replacement function in JS didn't look so great to me:
```
  // replace placeholder anchor tag markup in first tout to make link
  // add UTM params which are passed to FxA for account creation
  const fxaTout1 = fxaTouts[0].paragraph
  const url = getPreFxaUtmParams(serverUrl, 'create-account-link', userEmail)
  if ((/<a>/).test(fxaTout1) && (/<\/a>/).test(fxaTout1)) {
    const openingAnchorTag = `<a class="pre-fxa-nested-link" href="${url}" style="color: #0060df; font-family: sans-serif; font-weight: 300; font-size: 15px; text-decoration: none;">`
    fxaTouts[0].paragraph = fxaTout1.replace('<a>', openingAnchorTag)
  }
```

To add similar code for each instance would not be ideal.  I could look into an alternate solution if it's a concern though.  I agree, putting the `rel` in the string made me wince.

Reply:
It's a concern because some locales will translate the variable, others will translate attributes. 

While we have checks, fixing them one by one scales poorly (150 locales x 30 projects), so we should try to avoid any potential cause of breakage.

Indeed, that piece of code seems overly complex. Maybe it's possible to have a simpler, generic helper that only takes URL and extra attributes?

Reply:
@flodolo – updated strings to remove attributes from anchor tags 👍